internal/test-charm-repo/bundle/openstack/bundle.yaml

```diff
@@ -102,66 +102,66 @@ relations:
     - mysql:shared-db
   - - nova-cloud-controller:amqp
     - rabbitmq-server:amqp
-  - - nova-cloud-controller:image-service
```

Comment:
pretty sure those interface names correspond to openstack terms, not juju services/applications. simple sed script? :)